### ISSUE TITLE
SeedSigner core team dev time + new contributor onboarding

### ISSUE THREAD

**Issue by kdmukai** [TABConf/6.tabconf.com#77]
# Description
### What is this project about?
tldr: SeedSigner is a fully open-source bitcoin signing device that you can build yourself from off-the-shelf parts!

see: https://github.com/SeedSigner/seedsigner?tab=readme-ov-file#project-summary

### Is it a FOSS or Open Source project?
100% FOSS, MIT license.

The source code, the custom linux OS, and even the 3d printable enclosures.

### What would an attendee learn from visiting this projects table at Builder Days?
Anything ranging from:
* basic questions
* hands-on demos
* setting up an initial dev environment
* how to build our custom SeedSigner OS
* how to build custom images (i.e. specific branch targets)
* how to test PRs, recent merges, etc
* discuss possible new features or changes, etc.
* see the progress on the esp32 microntroller port
* ask Keith why the esp32 port is going so slowly...

### Relevant Links
GitHub Repo: 
[SeedSigner](https://github.com/SeedSigner)

### Who will run the table?
Most of the project's key leads will be there:

Project Creator / Lead: @seedsigner
Lead Maintainer: @newtonick (unconfirmed, but I hope!)
Lead Dev: @kdmukai
UI designer: @easyuxd
